config/api.yml

```diff
@@ -1762,6 +1764,14 @@
       :post:
       - :name: refresh
         :identifier: physical_chassis_refresh
+    :event_streams_subcollection_actions:
+      :get:
+      - :name: read
+        :identifier: event_streams_show_list
+    :event_streams_subresource_actions:
```

Comment:
I think this belongs under the `event_streams` as `subresource_actions`

Reply:
done

Reply:
done